fetch.sh

```diff
@@ -183,6 +185,24 @@ getwindowmanager () {
     fi
 }
 
+
+# Font (Configurable with "-F" and "--font" at launch)
+# Currently only supported for X Terminals
+if [ -f $HOME/.XResources ]; then
+    font=$(cat $HOME/.XResources | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+elif [ -f $HOME/.Xresources ]; then
+    font=$(cat $HOME/.Xresources | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+elif [ -f $HOME/.XDefaults ]; then
+    font=$(cat $HOME/.XDefaults | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+elif [ -f $HOME/.Xdefaults ]; then
+    font=$(cat $HOME/.Xdefaults | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+else
+    font="Unknown"
+fi
+
+# Screen Resolution (Configurable with "-R" and "--resolution" at launch)
+resolution=$(xdpyinfo | awk '/dimensions:/ {print $2}')
```

Comment:
This info is also highly innacurate for multi monitor setups. (It shows every monitor stitched together).

We should also think of an easy way for the user to hide specific info that they don't want to see. Something like this could work: 

```sh
    fetch.sh --hide "Packages Memory, Song"
```
We could also have some optional info lines that need to be enabled with a ```--show``` flag similar to above. This way we can include stuff like Xresources fonts etc.